jvb/src/main/kotlin/org/jitsi/videobridge/rest/binders/ServiceBinder.kt

```diff
@@ -29,5 +33,7 @@ class ServiceBinder : AbstractBinder() {
         bind(videobridgeSupplierSingleton).to(VideobridgeSupplier::class.java)
         bind(statsManagerSupplierSingleton).to(StatsManagerSupplier::class.java)
         bind(clientConnectionImplSupplierSingleton).to(ClientConnectionImplSupplier::class.java)
+        bind(healthCheckServiceSupplierSingleton).to(HealthCheckServiceSupplier::class.java)
+        bind(singleton).to(VersionServiceSupplier::class.java)
```

Comment:
Rename `singleton` to `versionServiceSupplierSingleton`?

Reply:
Yup, missed this one.  I've also since discovered that the bare name `singleton` isn't _great_ as you can then only have one per package in kotlin.  I'll either rename the import here or rename the singleton instances (the only reason I didn't rename the singleton instances from the get-go is that it's awkward from Java since you have to do `VersionServiceSupplierKt.versionServiceSupplierSingleton`)